tests/tests_integration/test_api/test_simulators/test_runs.py

```diff
@@ -10,21 +9,14 @@ def test_list_filtering(self, cognite_client: CogniteClient, seed_resource_names
         routine_external_id = seed_resource_names["simulator_routine_external_id"]
         runs_filtered_by_status = []
         for current_status in ["running", "success", "failure"]:
-            created_runs = cognite_client.simulators.runs.create(
-                [
-                    SimulationRunWrite(
-                        run_type="external",
-                        routine_external_id=routine_external_id,
-                    )
-                ]
-            )
+            created_run = cognite_client.simulators.routines.run(routine_external_id=routine_external_id)
```

Comment:
`cognite_client.simulators.runs.create` supports multiple items (by firing multiple request), and this one is only one per call.
do we want to support multiple here?

Reply:
Just as Functions do not allow to trigger multiple calls in a single method, I think we could follow the same pattern here: https://cognite-sdk-python.readthedocs-hosted.com/en/latest/functions.html#call-function

If someone wants to create multiple, they need to be mindful of that and handle the logic.

It's a tradeoff of simplicity for newcomers, versus advanced feature for advanced usage. The previous pattern which required users to import `SimulationRunWrite` to trigger a simple simulation is too much verbose in my opinion.

Reply:
I gave it another thought after sleeping over it and it should be fine to have both options available in the SDK. One that is easy and intuitive for a newcomer and one that allows creating multiple runs in one go, since the SDK has support for this out of the box.